src/tests/9_message.test.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from "chai";
+import { Conversation } from "../db/models/conversation.model";
+import ConversationService from "../db/service/ConversationService";
+import MessageService from "../db/service/MessageService";
+
+describe("Message service can", () => {
+	let conversationService: ConversationService;
+	let mockedConversation: Conversation;
+	let messageService: MessageService;
+	it("be setup", async () => {
+		conversationService = new ConversationService();
+		messageService = new MessageService();
+		expect(messageService).not.equal(undefined);
+		let response: any;
+		try {
+			response = await conversationService.createConversation(["user1", "user2"]);
+		} catch (error) {
+			console.log(error);
+		}
+		mockedConversation = response.conversation;
+	});
+
+	it("create a new message", async () => {
+		let response: any;
+		try {
+			response = await messageService.createMessage("user1", "abcdef", mockedConversation._id);
+		} catch (error) {
+			console.log(error);
+		}
+		expect(response).to.have.property("message");
+		expect(response.status).equals(200);
```

Comment:
```suggestion
		expect(response.status).equals(201);
```
create response returns `201`